documentation/docs/configuration/configTypes_tokenPermissions.md

```diff
@@ -6,68 +6,68 @@ sidebar_position: 7
-| notification                    | _/api/config/v1/notifications_                  | `Read Configuration` & `Write Configuration`                                                                        |
-| reports                         | _/api/config/v1/reports_                        | `Read Configuration` & `Write Configuration` |
-| request-attributes              | _/api/config/v1/service/requestAttributes_      | `Read Configuration` & `Capture request data`                                                                       |
-| request-naming-service          | _/api/config/v1/service/requestNaming_          | `Read Configuration` & `Write Configuration`                                                                        |
-| slo                             | _/api/v2/slo_                                   | `Read SLO` & `Write SLOs`                                                                                           |
-| service-detection-full-web-request   | _/api/config/v1/service/detectionRules/FULL_WEB_REQUEST_                 | `Read Configuration` & `Write Configuration`                                          |
-| service-detection-full-web-service   | _/api/config/v1/service/detectionRules/FULL_WEB_SERVICE_                 | `Read Configuration` & `Write Configuration`                                          |
-| service-detection-opaque-web-request | _/api/config/v1/service/detectionRules/OPAQUE_AND_EXTERNAL_WEB_REQUEST_  | `Read Configuration` & `Write Configuration`                                          |
-| service-detection-opaque-web-service | _/api/config/v1/service/detectionRules/OPAQUE_AND_EXTERNAL_WEB_SERVICE_  | `Read Configuration` & `Write Configuration`                                          |
-| service-resource-naming <br /> **SINGLE CONFIGURATION ENDPOINT**| _/api/config/v1/service/resourceNaming_                     | `Read Configuration` & `Write Configuration`                                          |
-| synthetic-location              | _/api/v1/synthetic/locations_                   | `Access problem and event feed, metrics, and topology` & `Create and read synthetic monitors, locations, and nodes` |
-| synthetic-monitor               | _/api/v1/synthetic/monitors_                    | `Create and read synthetic monitors, locations, and nodes`                                                          |
+| Configuration                                                                | Endpoint                                                                   | Token Permission(s)                                                                                                 |
+|------------------------------------------------------------------------------|----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
```

Comment:
Did you align that with some form of auto-formatting? 
Keeping that aligned with any added longer description, etc. seems tedious and like it will be off again eventually.

Reply:
Yes, and tbh I was not able to prevent it easily ;) 
At least Intellij does format it by default automatically. I would not force it to be aligned, but I think it is nice if we have it during editing.